test/quickstart.sh

```diff
@@ -87,9 +87,21 @@ wait_for_pod_ready model=gemma2-2b-cpu
 
 curl -s -X GET "http://localhost:8000/openai/v1/models" | jq '. | length == 4'
 
-curl http://localhost:8000/openai/v1/completions \
-  -H "Content-Type: application/json" \
-  -d '{"model": "gemma2-2b-cpu", "prompt": "Who was the first president of the United States?", "max_tokens": 40}'
+function test_completion() {
+  local url="$1"
+  http_code=$(curl -sw '%{http_code}' ${url} \
+    -H "Content-Type: application/json" \
+    -d '{"model": "gemma2-2b-cpu", "prompt": "Who was the first president of the United States?", "max_tokens": 40}')
+  if [ "$http_code" -ne 200 ]; then
+    echo "Failed to get completions from $url"
+    echo "HTTP code: $http_code"
+    exit 1
+  fi
+}
+
+test_completion "http://localhost:8000/openai/v1/completions"
+# Double slash in the URL should work as well.
+test_completion "http://localhost:8000/openai//v1/completions"
```

Comment:
This fails with 'HTTP code: 301'

You can use a curl param to follow the redirect or just assert the 301 with the correct target
```
-L, --location
              (HTTP) If the server reports that the requested page has moved to a different location (indicated with a Location: header and a 3XX response code), this option
              makes curl redo the request on the new place. If used together with -i, --include or -I, --head, headers from all requested pages are shown.
```

Reply:
Thanks for catching this! This now correctly follows and shows the 400 error.

Reply:
weird, on my local curl I can reproduce 400 error, but seems the automated test does not.